src/app/[entity]/[repo]/discussions/page.tsx

```diff
@@ -1,7 +1,33 @@
+// TODO: turn all of those imported scripts into more lightweight react components
+// change 503941a9939a4337d9aef7b92323c353441cb5ebe79f13fed77aeac615116354 to the correct event (once posted)
+
+import Script from 'next/script'
+
 export default function RepoDiscussionsPage() {
   return (
     <>
-      <h1>Discussions</h1>
+      <Script src="https://nostrocket.org/verifyBitcoinSignedMessage.js" />
+      <Script src="https://unpkg.com/nostr-tools/lib/nostr.bundle.js" />
+      <Script src="https://nostrocket.org/nostr.js" />
+      <Script src="https://nostrocket.org/comments.js" />
```

Comment:
we have to eventually turn this into proper react components

there is a lot of JS fluff in there which can be simplified

Reply:
yeah i agree, maybe in this PR?

Reply:
not sure, feel free to follow up

Reply:
> yeah i agree, maybe in this PR?

I have a lot of issues with the imported scripts. maybe we should consider just making it properly with react and the nostr functions we have already

Reply:
I thought I could get a quick win but it seems to cause more issues getting this to work than rebuilding it from scratch.

here is the thread: 

note10488mc62vzcf7yw8x0pavewdh5vmm5j4r279l9x43rss8tjzdc0s2lsrfp

https://iris.to/note10488mc62vzcf7yw8x0pavewdh5vmm5j4r279l9x43rss8tjzdc0s2lsrfp


let's try to display it in discussions and move all of our discussions there

Reply:
@pierresisson and @e-roy wanna help refactor this PR into proper react components and use

`nostr-tools` instead?

Reply:
we can get a lot of inspiration from here: https://nostrocket.org/comments.js -- should be easy to turn into a react component for us

Reply:
> @pierresisson and @e-roy wanna help refactor this PR into proper react components and use
> 
> `nostr-tools` instead?

I don't mind building out the UI, I just don't know how to get the data

Reply:
Not sure this will be useful or not but I updated https://nostrocket.org/comments.js recently. I think the new js file is less messy that before in handling thread. And better to convert to js. :) sorry im just a js pleb

Reply:
Hi @pqingshuang, thanks for the update :+1: I'll look into it.